packages/components/Molecules/Card/CardPromo.vue

```diff
@@ -38,15 +38,20 @@ export default {
   name: 'RplCardPromo',
   mixins: [formatdate, card],
   props: {
-    image: [String, Object],
-    date: [String, Object],
-    variation: String,
+    image: Object,
+    dateStart: String,
+    dateEnd: String,
     tag: String,
     topic: String,
     status: String,
     title: String,
     summary: String,
-    link: Object
+    link: Object,
```

Comment:
Does link a mandatory field for card? If it does, should we have a validation?

Reply:
I probably need to review this props declaration e.g. provide default values if it's not mandatory and explicitly declare required fields.